packages/flutter_tools/lib/src/commands/build_flx.dart

```diff
@@ -12,6 +12,7 @@ import 'build.dart';
 class BuildFlxCommand extends BuildSubCommand {
   BuildFlxCommand({bool verboseHelp: false}) {
     usesTargetOption();
+    usesPackagesOption();
```

Comment:
The reason why I put it in flutter_command.dart is because that allows me to skip the step that verifies whether a pubspec.yaml file is present. The other way to do that is to set a custom packages path in the PackageMap class: would it be better?